front_end/models/timeline_model/TimelineModel.ts

```diff
@@ -1737,6 +1737,26 @@ export enum RecordType {
   Coherent_CustomAttributeUpdate = 'Coherent_CustomAttributeUpdate',
   Coherent_CustomAttributeDeinit = 'Coherent_CustomAttributeDeinit',
   Coherent_Disabled = 'Coherent_Disabled',
+  Coherent_ProcessFrontendCommands = 'Coherent_ProcessFrontendCommands',
+  Coherent_ProcessLayer = 'Coherent_ProcessLayer',
+  Coherent_BatchCommands = 'Coherent_BatchCommands',
+  Coherent_DistributeLayers = 'Coherent_DistributeLayers',
+
+  Coherent_SubmitGlyphs = 'Coherent_SubmitGlyphs',
+  Coherent_SubmitManagerChanges = 'Coherent_SubmitManagerChanges',
+  Coherent_ProcessFrontendCommandsOnly = 'Coherent_ProcessFrontendCommandsOnly',
+  Coherent_DrawSDFGlyphs = 'Coherent_DrawSDFGlyphs',
+  Coherent_DrawSubLayerWithShaderFilter = 'Coherent_DrawSubLayerWithShaderFilter',
+  Coherent_Backend_Execute = 'Coherent_Backend_Execute',
```

Comment:
Not a big deal, but every other event **doesn't** have an underscore in the name, what's up with this one?